### DISCUSSION TITLE
Atri Engine gives an error when you click on Build and Run in development mode

### DISCUSSION THREAD

**Discussion by adityaxpique** [Atri-Labs/atrilabs-engine#217] [Help: Miscellaneous] [answered]
![Screenshot 2022-09-22 214057](https://user-images.githubusercontent.com/56078748/191801374-420be7b5-9110-4559-9e9b-61f3553b77dd.png)

Hello contributors, hope you are having a nice day! At this point of time we were not expecting open source contributors to run Build & Run  in development mode, hence, we did not add the steps in the `CONTRIBUTING.md`. Also we did not want to overload contributors with more steps. If you want to test out our framework in it's full glory please visit [docs.atrilabs.com](url)

**Comment by darshitac11** [comment, marked as answer]
`Contributing.md` has now been updated to include guidelines on how to create an Atri app. If you follow those instructions, you will not get the above error. 

Here's the [link](https://github.com/Atri-Labs/atrilabs-engine/blob/main/CONTRIBUTING.md#creating-an-atri-app) for your reference.
